lib/diagnostic.rb

```diff
@@ -38,24 +38,27 @@
 
 # define your method here
 
-Diagnostic.predicate = # call your method here
```

Comment:
Correct in using odd? for this, but we were asking you to define your own method that would check for this, such as:

```
def odd?(number)
  number % 2 == 1
end

Diagnostic.predicate = odd?(4)
```